public/index.html

```diff
@@ -62,10 +62,10 @@
     />
     <!-- OG tags require absolute url for images -->
     <meta name="twitter:image" content="https://excalidraw.com/og-image.png" />
-    <meta
+    <!-- <meta
```

Comment:
Need to revert before committing

Reply:
oh.. I disabled in dev, and committed by mistake :)